handout/components/app_structure/access_child_components_from_template.md

```diff
@@ -0,0 +1,38 @@
+# Access Child Components From the Template
+In our templates, we may find ourselves needing to access values provided by the child components which we use to build our own component.
```

Comment:
The first time that I read this paragraph I thought you were going to show the `@Output` decorator. I think it would be better to talk about "component's instance" instead of "component's values" when referring to template variables.

What about something about these lines:

_Oftentimes, we may find ourselves needing to access an instance of our child components which we use in other parts of our template._